src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -744,6 +800,7 @@ private void replaceEC2Servers() {
             ), e);
             return Collections.EMPTY_LIST;
         }
+        status.message = "Requesting instance reservation";
```

Comment:
This is a somewhat cryptic message. Since we have users that may not be super familiar with AWS lingo, it might be useful to change this to something like `String.format("Starting up %d new OTP instance(s)", count)`.